lib/arjdbc/vertica/adapter.rb

```diff
@@ -13,6 +13,19 @@ def self.sequence_name
     end
 
     def self.bulk_insert(columns, data)
+      unless columns.include?("created_at")
```

Comment:
Can we add a comment here explaining why this needs to be added now?

Reply:
After speaking with Mr. Deadsnake, it seems that in order to see why this was added initially, we'll need to pull the code out and re-test it to see what errors we encounter -- as it's been some time.  I don't mind going that route if needed.

Reply:
just as a point of order, it's mr dead sake (no n), 😂  the name is an anagram of derek adams